src/Automation/Automation/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Fix the issue of processing `PSCustomObject` and `Array`.
```

Comment:
Is there github issue id? Please use past tense.

Reply:
This is an ICM issue, should I create a github issue?

Reply:
No. let's mitigate IcM ticket too.